hack/preserve-bundle-labels.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -exou pipefail
+
+KUEUE_OPERAND_IMAGE="quay.io/redhat-user-workloads/kueue-operator-tenant/kubernetes-sigs-kueue@sha256:ec6688b483a2919d10d7ff6876f559c7b98961c6da425d80a653f0fe55db684d"
```

Comment:
Could we consider a different approach?

Maybe read from an environment variable?

This seems difficult to support programaticaly.

Reply:
I think `make bundle-generate` used to populate a `mustchange` value for all images. I believe we used to manually change them unless I'm missing something.

Reply:
My workload was to use `.envrc` and use something like `IMAGE_BUNDLE make bundle-generate`.